src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/ClientCredentialTypeTests.cs

```diff
@@ -6,8 +6,9 @@
 using System.ServiceModel.Security;
 using System.Text;
 using Xunit;
+using Infrastructure.Common;
 
-public static class Https_ClientCredentialTypeTests
+public class Https_ClientCredentialTypeTests : ConditionalWcfTest
```

Comment:
Reviewers: ConditionalFactDiscover only works with members of the type in which it appears, including base types or nested types.  You can't specify an arbitrary helper type.  So we either need to use derivation like this or duplicate the helper methods in every test class where we use them.